src/main/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -55,13 +55,13 @@
     private String likelihoodsAsString_PLs = null;
 
     /**
-     * initialize num likelihoods cache
+     * initialize num likelihoods cache to up to 5 alleles and 10 ploidy
      */
     static {
```

Comment:
This warm up should be moved into the constructor of the cache instead of a static block now that it has it's own place to live.

Reply:
Yeah, that's definitely more organized.  I'll move it